ts/Dashboards/Layout/Cell.ts

```diff
@@ -590,6 +590,21 @@ class Cell extends GUIElement {
         }
     }
 
+    /**
+     * Enables or disables the loading indicator in the cell.
+     *
+     * @internal
+     */
+    public setLoadingState(enabled: boolean = true): void {
+        const cell = this;
+        if (cell.container) {
```

Comment:
The cell.container is always available (I committed it a few days ago). Please check if `cell.container.classList` is fine, otherwise `cell.container?.classList` will be enough.